libs/crypto/include/crypto/mcl_dkg.hpp

```diff
@@ -31,24 +31,42 @@
 namespace bn = mcl::bn256;
 
 namespace fetch {
-namespace dkg {
+namespace crypto {
+namespace mcl {
+
+using PrivateKey     = bn::Fr;
```

Comment:
Might be good to use these in beacon_manager.hpp

Reply:
Sure.